src/main/resources/swagger/api-docs.yaml

```diff
@@ -2704,6 +2704,39 @@ paths:
         500:
           description: Internal Error
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/submissionsCount:
+      get:
+        responses:
+          '200':
+            description: Successful Request
+            schema:
+              type: object
+              description: Map[String,Int]
```

Comment:
The model & example fields are empty in swagger-ui because swagger only knows this as an `object`, and all it can render is `{}`.  To populate it, you'd have to create something in `definitions` like:
```
  SubmissionCount:
    type: object
    properties:
      Accepted:
        type: integer
      Evaluating:
        type: integer
      Submitting:
        type: integer
      Submitted:
        type: integer
      Aborting:
        type: integer
      Aborted:
        type: integer
      Done:
        type: integer
```
and then reference it from here with:
```
          '200':
            description: Successful Request
            schema:
              $ref: '#/definitions/SubmissionCount'
```
While we put effort into documenting Rawls, since Rawls isn't public-facing (yet), we've cut some corners. Since orchestration is our public-facing API, we should do our best to make doc comprehensive.